config/schema/strawberryfield.schema.yml

```diff
@@ -79,9 +79,17 @@ strawberryfield.storage_settings:
     file_scheme:
       type: string
       label: 'Storage Scheme for Persisting Files'
+    file_path:
```

Comment:
Ok, do we need both? if we add here the file_path then the alter hook needs to be adapted again? Just wondering since the ISSUE was about the Object File Path. But happy to deal with both (we need an alter too in the form)

Reply:
Yes, this is not strictly necessary since we can modify the path through hook_strawberryfield_file_destination_alter(). However, it seems to me that being able to set the path would be desired pretty often, and requiring a user to implement a hook just to set this to a different path seems a bit onerous, particularly as they will see that we do give them the configuration option for the digital object file storage.
I don't feel strongly about this though, Happy to take it out if you think that makes more sense.

Reply:
Ok, let's keep it. I understand how this can be useful for vanilla implementations!